### PR TITLE
Progess bar replacement

### PR DESCRIPTION
**Pull request by Deepakanandrao**
Css replacement for stats

### CODE REVIEW

**Comment by Deepakanandrao** [comment]
@stephsteph123 check out the files section for changes and add a couple of comments

File: index.css [hunk 1]
```diff
@@ -351,50 +361,51 @@ aside ul li img {
...
-    display: flex;
-    flex-direction: column;
-    justify-content: space-around;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
 }
 
-.results-left-1 {
-    display: flex;
```
Comment (Deepakanandrao) [inline index.css:378]: Red section - Code I removed

File: index.css [hunk 2]
```diff
@@ -351,50 +361,51 @@ aside ul li img {
...
-    padding: 4px;
-}
+.row {
+  display: flex;
+  padding: 4px;
```
Comment (Deepakanandrao) [inline index.css:389]: Green - Code I've Added

File: index.css [hunk 3]
```diff
@@ -351,50 +361,51 @@ aside ul li img {
...
 }
 
 .results-card {
-    display: flex;
-    flex-direction: column;
-    justify-content: space-between;
-    background-color: white;
-    padding: 16px;
-    border-radius: 16px;
```
Comment (Deepakanandrao) [inline index.css:363]: Sometimes it's just formatting issue like here - only spaces are removed

**Comment by Deepakanandrao** [comment]
@stephsteph123  If everything looks good and acceptable, 
the convention is to give a 'thumbs up' by saying LGTM (looks good to me) and approve the PR. 
Then the PR creator merges the PR

**Review by stephsteph123 — COMMENTED** [review]
looks good to me

**Pull request merged**